controller.go

```diff
@@ -392,7 +397,7 @@ func newCronJob(apprepo *apprepov1alpha1.AppRepository) *batchv1beta1.CronJob {
 func newJob(apprepo *apprepov1alpha1.AppRepository) *batchv1.Job {
 	return &batchv1.Job{
 		ObjectMeta: metav1.ObjectMeta{
-			GenerateName: cronJobName(apprepo) + "-",
+			GenerateName: cronJobName(apprepo),
```

Comment:
AFAIK GenerateName should have a trailing dash as it will append a random string

Reply:
nope, it leaves it as you set it:
```
- apiVersion: batch/v1
  kind: Job
  metadata:
    creationTimestamp: 2018-02-05T17:50:46Z
    generateName: apprepo-stable-
    labels:
      apprepositories.kubeapps.com/repo-name: stable
      controller-uid: 17ddc5be-0a9d-11e8-9145-08002714fe69
      job-name: apprepo-stable-t7nbq
    name: apprepo-stable-t7nbq
```